<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_form">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Форма регистрации</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="text">
     <string>Имя пользователя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pasport">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Паспорт:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пол:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_fem">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Женский</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_male">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Мужской</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_registration">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Сохранить данные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openFile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
